### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by enigmaoftech** [advplyr/audiobookshelf#1210] [Ideas]
It would be nice to have the ability to use an idp for authentication into the service. Using something like authentik for the users and letting them SSO into the app.

**Comment by Guruleenyc** [comment]
Bump! I'd like to use Plex or social media SSO

**Comment by Guruleenyc** [comment]
I'd love to get Authentik working with this project.

**Comment by hiro24** [comment]
bumping as well. I'd love to get SSO working w/ something like authentik.

**Comment by MadDogTen** [comment]
Looks like this is being worked on with PR #1636. Hopefully everything works out, Looking forward to it!

**Comment by nyhil116** [comment]
It's been about 4 months and no news :( 
An SSO solution for this app would be incredible tho.. I'll hold out hope!

**Reply by alexanderadam** [reply]
It's surely not a solution for everybody but there's also PodFetch which has [Podcast-wise similar features like Audiobookshelf](https://github.com/SamTV12345/PodFetch/tree/main#roadmap), [a nice UI](https://github.com/SamTV12345/PodFetch/blob/main/docs/UIWalkthrough.md#ui) and uses less resources. And [PodFetch supports SSO via OpenID Connect](https://github.com/SamTV12345/PodFetch/blob/main/docs/AUTH.md#oidc) (see also [issue 35](https://github.com/SamTV12345/PodFetch/issues/35)) already.
